### PR TITLE
add vae-gru and cleanup

### PR DESCRIPTION
**Pull request by hubertrybka**
- add new architectures
- Create no-teacher-0.pt
- add better loss function
- add teacher forcing to _v2 and _v3
- add image logging to wandb
- minor image logging fix
- minor fix v2
- add better loss function to training
- fix loss function in evaluate
- gru_v3_improvement
- Slight change
- .
- Update gru_v2.py
- fix teacher forcing
- fix evaluate
- fix teacher forcing again
- minor fix
- fix teacher forcing (once again)
- Update GRU_MI.ipynb
- Create v2_w-teacher-0
- Update GRU_MI.ipynb
- add smiles enumeration
- minor fix
- review dataset
- quick fiq
- minor fix
- Create v3_w-teacher-w-enumeration-0
- Update GRU-smiles-enumerator.ipynb
- Create results_visualisation.ipynb
- move alphabet outside the notebook
- Update chembl_prep.ipynb
- Update chembl_prep.ipynb
- Update chembl_prep.ipynb
- Update .gitignore
- Update .gitignore
- move old data files elsewhere
- delete old datasets
- add scaffold split
- minor fix
- rename
- Update .gitignore
- add train and test to .parquet
- minor fix
- Update results_visualisation.ipynb
- Exceptions added
- add exception handling
- add auto parameters saving
- add mkdir for each new model
- minor fix
- Update example_printer.py
- fix example printer
- minor fix in example printer
- fix whatever was wrong with train
- minor fix
- minor fix
- wahtever at this point
- GRU_v4
- Example Printer fix & New Architecture
- fix vectorization
- fix interactive mol view
- add fragments visualisation
- add comments
- add Highlighter
- add manual
- fix Highlighter class
- add multiple molecules generation
- add shell script
- fix script
- Delete enviroment.yml
- move
- Create enviroment.yml
- Update train_gru.ssh
- Update train_gru.ssh
- Update train_gru.ssh
- Update train_gru.ssh
- update script
- Update train_gru.ssh
- Update train_gru.ssh
- Update train_gru.py
- change wandb login
- update train script
- update train script
- Update train_gru.py
- CVAE->VAE
- Update train_gru.py
- Update train_gru.py
- Update train_gru.py
- Update train_gru.py
- Update train_gru.py
- Update train_gru.ssh
- comment out wandb commands
- Update train_gru.py
- Update train_gru.py
- Update train_gru.py
- Update train_gru.py
- rename classes and modules
- add validation
- remove useless code
- fix dataset path
- quick fix
- prepare training script
- change gru num_workers
- Update train_gru.py
- Update train_vae.py
- Update train_vae.py
- prepare for sweep
- Update train_vae.py
- quick fix
- prepare run
- add vae+gru architecture
- add vae and gru model
- prepare for run
- quick fix
- prepare for run
- Compounds Filter
- remove old notebooks
- add ring size control
- remove ring control
- Update results_visualisation.ipynb
- Update vae_gru.py
- Filter update
- Filter update
- Update .gitignore
- add trained models
- add fp sampler
- name change
- prepare for run
- fix cross-entropy
- prepare for training
- whoooop
- repo cleanup
- repo cleanup
- Update results_visualisation.ipynb

### CODE REVIEW

**Comment by hubertrybka** [comment]
Bug in train_gru.ssh
